opds.py

```diff
@@ -870,16 +870,22 @@ def add_breadcrumb_links(self, lane, entrypoint=None):
         self.show_current_entrypoint(entrypoint)
 
     @classmethod
-    def search(cls, _db, title, url, lane, search_engine, query, media=None, pagination=None,
-               annotator=None, languages=None, facets=None
```

Comment:
It's no longer necessary to pass in `media` or `languages`, because that's now handled by either `lane` or `facets`.